src/pages/Defi/hooks/useFoxyBalances.tsx

```diff
@@ -25,9 +25,10 @@ export function useFoxyBalances() {
 
   useEffect(() => {
     ;(async () => {
-      const chainAdapter = await getChainAdapterManager().get(KnownChainIds.EthereumMainnet)
+      const chainAdapter = getChainAdapterManager().get(KnownChainIds.EthereumMainnet)
       if (!chainAdapter || !wallet) return
-      const userAddress = await chainAdapter.getAddress({ wallet })
+      const bip44Params = chainAdapter.getBIP44Params({ accountNumber: 0 })
```

Comment:
similarly big todo to remove this

Reply:
Captured in https://github.com/shapeshift/web/issues/2730